### ISSUE TITLE
App crashing on toggling the attendee checkin

### ISSUE THREAD

**Issue by sridharjajoo** [fossasia/open-event-orga-app#1019]
### Bug Report    

**Actual Behaviour**
Currently suppose we toggle the attendee checked-in or checked-out and subsequently try to do the same for other attendees as well, the app crashes mid way.

**Steps to reproduce it**
Check in a few attendees, the app will crash mid way.

**LogCat for the issue**
```
06-06 00:06:06.886 11526-11526/? E/subsystem_ramdump: Unable to open /dev
    Failed to find pil ramdump
06-06 00:06:08.839 11429-11530/org.fossasia.eventyay E/WorkerWrapper: Didn't find WorkSpec for id 5c999bd4-0703-409d-a778-396a7e169abc
06-06 00:06:09.150 3751-4332/? E/HAL: hw_get_module_by_class: module name gralloc
    hw_get_module_by_class: module name gralloc
06-06 00:06:09.158 3043-3043/? E/Icon: Unable to load resource 0x00000000 from pkg=com.android.systemui
    android.content.res.Resources$NotFoundException: Resource ID #0x0
        at android.content.res.Resources.getValue(Resources.java:1369)
        at android.content.res.MiuiResources.getValue(MiuiResources.java:146)
        at android.content.res.Resources.getDrawable(Resources.java:822)
        at android.graphics.drawable.Icon.loadDrawableInner(Icon.java:313)
        at android.graphics.drawable.Icon.loadDrawable(Icon.java:269)
        at android.graphics.drawable.Icon.loadDrawableAsUser(Icon.java:377)
        at com.android.systemui.statusbar.ExpandedIcon.getDrawable(ExpandedIcon.java:59)
        at com.android.systemui.statusbar.StatusBarIconView.getIcon(StatusBarIconView.java:179)
        at com.android.systemui.statusbar.StatusBarIconView.setIcon(StatusBarIconView.java:136)
        at com.android.systemui.statusbar.StatusBarIconView.updateDarkMode(StatusBarIconView.java:271)
        at com.android.systemui.statusbar.phone.SimpleStatusBar.updateDarkMode(SimpleStatusBar.java:275)
        at com.android.systemui.statusbar.phone.PhoneStatusBar$42.run(PhoneStatusBar.java:3862)
        at android.os.Handler.handleCallback(Handler.java:742)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:157)
        at android.app.ActivityThread.main(ActivityThread.java:5571)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:745)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:635)
```
**Android version and Phone Model**
Lollipop (5.1.1) and Redmi Note 3

**Would you like to work on the issue?**
Yes

**Screenshots**
![ezgif com-resize 4](https://user-images.githubusercontent.com/20684618/40996045-7f91e52e-691e-11e8-9033-9ef8ee60d414.gif)

**Comment by sridharjajoo** [comment]
@iamareebjamal I think it is because of this ```Didn't find WorkSpec for id 5c999bd4-0703-409d-a778-396a7e169abc```

**Comment by iamareebjamal** [comment]
I think it is due to this Unable to load resource 0x00000000 from pkg=com.android.systemui
    android.content.res.Resources$NotFoundException: Resource ID #0x0

**Comment by sridharjajoo** [comment]
The logs above were there when the app was tested on a xiaomi redmi note 3 device.
The ``unable to load resource`` was not present when the app was tested on a SAMSUNG device but the logs still had ```Didn't find WorkSpec for id 5c999bd4-0703-409d-a778-396a7e169abc```

**Comment by iamareebjamal** [comment]
OK

**Comment by Masquerade0097** [comment]
I tried checking attendees in and out repeatedly on Moto G4 Plus and got two different errors - 

1. `Didn't find WorkSpec for id a02d3266-9003-4ecb-8e59-540ff811a882` (Same as you mentioned in this issue)
2. `java.lang.NullPointerException: Attempt to invoke virtual method 'boolean androidx.work.State.isFinished()' on a null object reference`

On researching, I found out that error no 2 is present in WorkManager version 1.0.0-aplha01. The next version alpha02 resolves it. I've created a separate issue for this.

But even on upgrading the version error no 1 is still there.

**Comment by sridharjajoo** [comment]
@Masquerade0097 Actually I had also got the second error and I was going to update the WorkManager version in the P.R

**Comment by Masquerade0097** [comment]
@sridharjajoo Since you didn't mentioned it in the issue details or logcat therefore I thought you didn't get that error. So I created a new Issue for that and sent PR. Should I close the PR and issue ?

**Comment by sridharjajoo** [comment]
@Masquerade0097 Do you know why the WorkSpec error is being caused?

**Comment by Masquerade0097** [comment]
@sridharjajoo As of now, I don't know the reason. Will look into it.

**Comment by iamareebjamal** [comment]
Status @sridharjajoo @Masquerade0097 @srv-twry @mishuvs

**Comment by Masquerade0097** [comment]
Resolved #1226
